extensions/measurement-tracking/src/panels/PanelMeasurementTableTracking/ActionButtons.tsx

```diff
@@ -2,31 +2,26 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { useTranslation } from 'react-i18next';
 
-import { Button, ButtonGroup } from '@ohif/ui';
+import { Button, ButtonSize, ButtonType } from '@ohif/ui';
 
 function ActionButtons({ onExportClick, onCreateReportClick, disabled }) {
   const { t } = useTranslation('MeasurementTable');
 
   return (
     <React.Fragment>
       <Button
-        className="text-base px-2 py-2"
-        size="initial"
-        variant={disabled ? 'disabled' : 'outlined'}
-        color="black"
-        border="primaryActive"
         onClick={onExportClick}
         disabled={disabled}
+        type={ButtonType.secondary}
+        size={ButtonSize.small}
```

Comment:
Hmmm, I was fine with the ButtonType to be exported, until here that I see ButtonSize is also another export. Could we group them all under `ButtonStyle` and then we have `ButtonStyle.type.secondary`, and `ButtonStyle.size.small`. Or anyother name that make it a group. I'm saying this since we will probably have this pattern of a UI component with its style Enums. Maybe even ButtonEnums

Reply:
Makes sense. I made changes... hopefully they are what you meant.